### PR TITLE
Privacy policy

### PR DESCRIPTION
**Pull request by kicferk1**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDEL-512

### Is there anything weird that code reviewer should know?
Nope, everything is straightforward.

### Review Checks
- [x] All commit messages are meaningful and true
- [x] Added enough automated tests
- [x] All forms have an `id` attribute on them

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Click on the 'Privacy' link in the footer.

### CODE REVIEW

File: app/views/privacy_policies/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    <h2 class="govuk-heading-l">How we use your data</h2>
+    <h3 class="govuk-heading-m">Tracking progress through self study materials</h3>
+    <p class="govuk-body">Tracking progress through self study materials includes the following:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Identify which self study materials are relevant to you</li>
+      <li>identifying which self study materials have been accessed</li>
+      <li>getting in touch if there’s a security issue concerning your data</li>
+      <li>getting in touch with candidates and schools about progress through self study materials</li>
+      <li>getting in touch with candidates and school about self study materials</li>
+      <li>Match, link and analyse your data along with other datasets</li>
```
Comment (xswoodsx) [inline app/views/privacy_policies/show.html.erb:52]: Some lines start with a capital and some don't. English isn't my strongest but could be intentional?

File: app/views/privacy_policies/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      If you want to ask us to remove your data or get access to the data we have about you, you can email us on:
+    </p>
+
+    <p>
+      <%= render MailToSupportComponent.new %>
+    </p>
+
+    <p class="govuk-body">
+      You can also use our contact form to
+      <% govuk_link_to "get in touch with our Data Protection Officer", "https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen" %>.
```
Comment (xswoodsx) [inline app/views/privacy_policies/show.html.erb:144]: Just needs the = in the <%

File: app/views/privacy_policies/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    </p>
+    <p class="govuk-body">
+      Once an early career teacher has enrolled with a teacher training provider, we will not be able to
+      ask them to remove personal data from their systems. Please contact the provider separately.
+    </p>
+    <p class="govuk-body">
+      For further information or to raise a concern, visit the
+      <% govuk_link_to "Information Commissioner’s Office", "https://ico.org.uk/" %> (ICO).
```
Comment (xswoodsx) [inline app/views/privacy_policies/show.html.erb:152]: Same as above

File: app/views/privacy_policies/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+      <li>access personal data</li>
+      <li>have incorrect data updated</li>
+      <li>have data erased</li>
+      <li>stop or restrict the processing of your data</li>
+      <li>get and reuse your data for different services</li>
+      <li>object to how your data is processed in certain circumstances</li>
+    </ul>
+    <p class="govuk-body">
+      You can find out more about how we handle personal data in our
+      <% govuk_link_to "personal information charter", "https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter" %>.
```
Comment (xswoodsx) [inline app/views/privacy_policies/show.html.erb:130]: As below, needs a =

File: app/views/privacy_policies/show.html.erb (hunk 2 above)
Comment (kicferk1) [reply app/views/privacy_policies/show.html.erb:144]: Good spot :)

File: app/views/privacy_policies/show.html.erb (hunk 1 above)
Comment (kicferk1) [reply app/views/privacy_policies/show.html.erb:52]: I copy pasted it from a content doc...
But I should probably double check this.

**Review by xswoodsx — APPROVED** [review]

File: app/views/privacy_policies/show.html.erb (hunk 1 above)
Comment (kicferk1) [reply app/views/privacy_policies/show.html.erb]: I think they should be lower case judging by other places. When Mark gets back to me I can push a copy fix, but I wanna get it in before I start working on accessibility statement.

**Pull request merged**